x-pack/test/apm_api_integration/basic/tests/transaction_groups/breakdown.ts

```diff
@@ -48,35 +48,79 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         expect(response.status).to.be(200);
         const { timeseries } = response.body;
         const { title, color, type, data, hideLegend, legendValue } = timeseries[0];
-        expect(data).to.eql([
-          { x: 1593413100000, y: null },
-          { x: 1593413130000, y: null },
-          { x: 1593413160000, y: null },
-          { x: 1593413190000, y: null },
-          { x: 1593413220000, y: null },
-          { x: 1593413250000, y: null },
-          { x: 1593413280000, y: null },
-          { x: 1593413310000, y: 1 },
-          { x: 1593413340000, y: null },
-        ]);
-        expect(title).to.be('app');
-        expect(color).to.be('#54b399');
-        expect(type).to.be('areaStacked');
-        expect(hideLegend).to.be(false);
-        expect(legendValue).to.be('100%');
+
+        expectSnapshot(data).toMatchInline(`
+          Array [
+            Object {
+              "x": 1593413100000,
+              "y": null,
+            },
+            Object {
+              "x": 1593413130000,
+              "y": null,
+            },
+            Object {
+              "x": 1593413160000,
+              "y": null,
+            },
+            Object {
+              "x": 1593413190000,
+              "y": null,
+            },
+            Object {
+              "x": 1593413220000,
+              "y": null,
+            },
+            Object {
+              "x": 1593413250000,
+              "y": null,
+            },
+            Object {
+              "x": 1593413280000,
+              "y": null,
+            },
+            Object {
+              "x": 1593413310000,
+              "y": 1,
+            },
+            Object {
+              "x": 1593413340000,
+              "y": null,
+            },
+          ]
+        `);
+
+        expectSnapshot({
+          title,
+          color,
+          type,
+          hideLegend,
+          legendValue,
+        }).toMatchInline(`
+          Object {
+            "color": "#54b399",
+            "hideLegend": false,
+            "legendValue": "100%",
+            "title": "app",
+            "type": "areaStacked",
+          }
+        `);
```

Comment:
Any advantages to grouping the values over doing separate expectations?

Reply:
Yeah, I prefer the latter approach. This was also just a first iteration that I neglected to update. I'll replace it.